### DISCUSSION TITLE
FRI中，如何选取blow up前的小域D与后的大域F，有什么讲究吗？

### DISCUSSION THREAD

**Discussion by 0xxEric** [coset-io/zkp-academy#47] [Q&A] [answered]
在STARK101中：
blow up前的小域D=<g>，g=5^{3*2^20}
blow up 后的大域F={w, wh, wh^2,......wh^8191}，其中w=generator=5，h=5{3*2^30 // 8192}

请问，
（1）为何这样取，这样取有什么讲究吗？（整个的域是F_(3 * 2**30 + 1)）
（2）只需要按，大域与小域隔开，不重叠就可以吗？

**Comment by readygo67** [comment, marked as answer]
1.  建议区和原来的D 不重叠的coset, 这样能避免构造cp 是出现除数是0的情况。
2. 能起到对原来的值盲化的作用。
